src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,31 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import { Todo } from '../../api/todos';
+import { User } from '../../api/users';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: Todo;
+  // user?: User;
+}
+
+export const TodoInfo: FC<Props> = props => {
+  const { todo} = props;
+  const { id, completed, title, userId } = todo;
+
+  const filteredUser = usersFromServer.find(
+    (user: User): boolean => user.id === userId,
+  );
```

Comment:
Prepare todos with users, and pass them as a prop. Presentational components shouldn't have any logic https://www.geeksforgeeks.org/reactjs-container-and-presentational-pattern-in-components/